src/utils/slicer.py

```diff
@@ -75,29 +71,26 @@ def __call__(self, text: str, max_slice_count: int = None) -> tuple[torch.Tensor
 class VTRSlicer:
     def __init__(
         self,
-        window_size: int = 25,
-        stride: int = 10,
-        font: str = "fonts/NotoSans.ttf",
-        font_size: int = 15,
+        char2array: dict,
+        window_size: int = 32,
+        stride: int = 5,
     ):
-        logger.info(f"Init VTRSlicer | window_size={window_size}, stride={stride}, font={font}, font_size={font_size}")
+        logger.info(f"Init VTRSlicer | window_size={window_size}")
         self.window_size = window_size
         self.stride = stride
-        self.font = font
-        self.font_size = font_size
+        self.char2array = char2array
 
     def __call__(self, text: str, max_slice_count: int = None) -> torch.Tensor:
-        image = text2image(text, font=self.font, font_size=self.font_size)
-        image_bytes = torch.as_tensor(np.array(image))
+        unknown_token = self.char2array["UNK"]
+        image_bytes = torch.Tensor(np.concatenate([self.char2array.get(char, unknown_token) for char in text], axis=1))
```

Comment:
`Tensor` is a type and not for direct usage, use `tensor` function to create it (https://pytorch.org/docs/stable/tensors.html#torch.Tensor)